### ISSUE TITLE
Release 0.2.0 on appspot using a tagged commit and hooks/automation

### ISSUE THREAD

**Issue by davelab6** [googlefonts/fontbakery#1151]
@felipesanches please set up a git hook or something such that your appspot instance is in sync with master

**Comment by felipesanches** [comment]
First step is figuring out whether simply re-running **gclod app deploy** is enough. (I think/hope it is!)

I'll try that now. And if it works, then I'll try to automate it.

**Comment by felipesanches** [comment]
ok, **gcloud app deploy** does its job (and pretty fast!)
I think I'll simply get used to manually running it after each important pull-request merge.

**Comment by felipesanches** [comment]
I looked up githooks documentation but it seems that it's not exactly what we would need here since the actual update on git master is performed by github itself (whenever I press the "merge pull request" button on their web ui) instead of being tied to specific git commands I run locally at my working copy.

The only interactions I have between my local copy and the origin remote (hosted on github) are:
* rebasing master (to incorporate previously merged commits)
* pushing feature-branches (prior to registering a corresponding pull request)

I could perhaps use Github Webhooks but it sounded too complicated for this usage scenario. I think it is easier to do deploy the webapp by hand (running **gcloud app deploy**) at least once a day, or after significant webapp fixes. I'll get used to it.

**Comment by davelab6** [comment]
Why not use travis to do this?

**Comment by felipesanches** [comment]
ok, cool! It makes sense. And it seems that this page explains how to do it. I'll try to set this up:

https://docs.travis-ci.com/user/deployment/google-app-engine/

**Comment by felipesanches** [comment]
I was able to generate a service api key in json format, then encrypt it with the travis command line and commit it (the encrypted file and its entry on .travis.yml) but the openssl command to decrypt the credentials server-side is failing on Travis.

Apparently the reason for that is that the credentials need to be encrypted specifically for the repository they're meant to be used at. I have used the felipesanches/fontbakery repo (my 'origin' remote) instead of the googlefonts/fontbakery repo (my 'upstream' remote).

I was not able yet to figure out how to select a different repo when encrypting the credentials with the travis command line tool. I see that its got a repos command, but it fails to give me any meaningful info. It spits out a cryptic message like this:

```
414: "" 
```

It also said the travis cli version is old and that I should update it, but the gem install procedure fails. I did a **gem2.0 install travis** and it seems to have succeeded, but the 414 problem did not go away yet after that.

I'd appreciate any advice people may have here because I have already spent too much time trying to figure this out. I'll move on to something else for a while and I'll revisit this issue later.
